src/styles/_description.scss

```
@@ -0,0 +1,24 @@
+.description {
+  position: absolute;
+  bottom: 165px;
+  color: #fff;
+  font-size: 60px;
+  line-height: 60px;
+  font-weight: bold;
+  letter-spacing: -1.5px;
```

```suggestion
.description {
  position: absolute;
  bottom: 165px;
  
  font-size: 60px;
  line-height: 60px;
  font-weight: bold;
  letter-spacing: -1.5px;
  
  color: #fff;
```
так буде читабельніше, розсортуй усі стилі відповідно до характеристик